app/services/foreman/plugin.rb

```diff
@@ -243,7 +243,9 @@ def pending_migrations
       pending_migrations = ActiveRecord::Migrator.new(:up, migration_paths).
         pending_migrations
 
-      pending_migrations.size > 0
+      return false unless pending_migrations.size > 0
+      Rails.logger.warn(
```

Comment:
Yeah, debug might work. You're right, they're definitely pending migrations that can be logged, I'm just wary of adding a _warning_ that will trigger under normal circumstances at installation.